invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/user_dashboard/components/requests.js

```diff
@@ -144,35 +145,34 @@ export function RequestsResultsItemTemplate({ result, index }) {
   const createdDate = new Date(result.created);
   const differenceInDays = timestampToRelativeTime(createdDate.toISOString());
   return (
-    <Item key={index}>
+    <Item key={index} className="community-item">
       <Item.Content>
         <Item.Header>
           {result.type && (
-            <span className="mr-5">
-              <Label size="large">{result.type}</Label>
-            </span>
+            <Label size="large" className="rel-mr-1">{result.type}</Label>
           )}
-          <a href={`/me/requests/${result.id}`}>{result.title}</a>
+          <a className="header-link" href={`/me/requests/${result.id}`}>{result.title}</a>
         </Item.Header>
 
-        <Item.Meta className="mt-10">
-          <span className="mr-15">
+        <Item.Meta>
+          <div className="inline-computer rel-mr-1 rel-mt-1 rel-mb-1">
             {/* TODO: Replace by resolved user */}
             {/* {i18next.t(`opened {{difference}} by {{user}}`, {
               difference: differenceInDays,
               user: result.created_by.user,
             })} */}
-            {`opened ${differenceInDays} by you`}
-          </span>
+            {i18next.t('Opened ') + differenceInDays + i18next.t(' by you')}
+          </div>
+
           {result.receiver.community && (
-            <>
+            <div className="inline-computer">
```

Comment:
```suggestion
            <div className="flex">
```
is this class doing the job?

Reply:
No, but the flex class could be extended to cover this case if that's a better solution.

Reply:
what do you mean by extended? Could you give more details ?
in case if it is too complicated we can leave it like this for now. 
I think the PR needs a rebase

Reply:
The flex class by itself is not enough to display the content correctly. It will have to be combined with sibling classes that make sure the content is breaking to a column for mobile (e.g. `.column-mobile`) and aligns correctly on desktop (e.g. `align-items-center` on the parent or `.align-self-center` to the children  - both of which will need adjustment for column view on mobile). Adding `.flex` to the parent does the same as having the children displaying as inline-blocks on all devices. Adding `.flex` to the children is the same as having them display as blocks.